usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -54,75 +57,92 @@ def signal_handler(signal, frame):
+            raise CommandError(f"Found multiple submissions with id {submission_id}")
 
-        submission_data = submission_data[0].copy()
+        submission_data = submission_data[0]
 
         if submission_data["publish_status_id"] not in (2, 3):
             raise RuntimeError(f"publish_status_id {submission_data['publish_status_id']} is not allowed")
         if submission_data["d2_submission"] is not False:
             raise RuntimeError(f"d2_submission {submission_data['d2_submission']} is not allowed")
 
-        db_cursor.execute(f"select exists(select from certify_history where submission_id = {submission_id})")
-        if db_cursor.fetchone()[0] is not True:
-            raise RuntimeError("Submission does not exist in certify_history")
-
         submission_attributes = get_submission_attributes(submission_id, submission_data)
 
         logger.info("Getting File A data")
         db_cursor.execute("SELECT * FROM certified_appropriation WHERE submission_id = %s", [submission_id])
         appropriation_data = dictfetchall(db_cursor)
         logger.info(
-            "Acquired File A (appropriation) data for "
-            + str(submission_id)
-            + ", there are "
-            + str(len(appropriation_data))
-            + " rows."
+            f"Acquired File A (appropriation) data for {submission_id}, there are {len(appropriation_data):,} rows."
         )
         logger.info("Loading File A data")
         start_time = datetime.now()
         load_file_a(submission_attributes, appropriation_data, db_cursor)
-        logger.info("Finished loading File A data, took {}".format(datetime.now() - start_time))
+        logger.info(f"Finished loading File A data, took {datetime.now() - start_time}")
 
         logger.info("Getting File B data")
         prg_act_obj_cls_data = get_file_b(submission_attributes, db_cursor)
         logger.info(
-            "Acquired File B (program activity object class) data for "
-            + str(submission_id)
-            + ", there are "
-            + str(len(prg_act_obj_cls_data))
-            + " rows."
+            f"Acquired File B (program activity object class) data for {submission_id}, "
+            f"there are {len(prg_act_obj_cls_data):,} rows."
         )
         logger.info("Loading File B data")
         start_time = datetime.now()
         load_file_b(submission_attributes, prg_act_obj_cls_data, db_cursor)
-        logger.info("Finished loading File B data, took {}".format(datetime.now() - start_time))
+        logger.info(f"Finished loading File B data, took {datetime.now() - start_time}")
 
         logger.info("Getting File C data")
         certified_award_financial = CertifiedAwardFinancial(submission_attributes)
         logger.info(
             f"Acquired File C (award financial) data for {submission_id}, "
-            f"there are {certified_award_financial.count} rows."
+            f"there are {certified_award_financial.count:,} rows."
```

Comment:
Typo? `count:,` i see this a few times. This must be something pythonic i don't know.

Reply:
Oh... maybe a format string? Saying put commas in the numbers